presto-product-tests/src/main/java/com/facebook/presto/tests/utils/JdbcDriverUtils.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests.utils;
+
+import java.sql.Connection;
+
+public class JdbcDriverUtils
+{
+    public static String getSessionProperty(Connection connection, String key) throws SQLException
+    {
+        try (Statement statement = connection.createStatement()) {
+            ResultSet rs = statement.executeQuery("SHOW SESSION");
+            while (rs.next()) {
+                if (rs.getString("Name").equals(key)) {
+                    return rs.getString("Value");
+                }
+            }
+        }
+        return null;
+    }
+
+    public static String getSessionPropertyDefault(Connection connection, String key) throws SQLException
+    {
+        try (Statement statement = connection.createStatement()) {
+            ResultSet rs = statement.executeQuery("SHOW SESSION");
+            while (rs.next()) {
+                if (rs.getString("Name").equals(key)) {
+                    return rs.getString("Default");
```

Comment:
these two methods differ with one String. Maybe extract a method which takes this String as parameter. So this function would look like:

```
public static String getSessionPropertyDefault(Connection connection, String key) throws SQLException
{
    return getSessionProperty(connection, key, "Default");
}
```